LayoutTests/imported/w3c/web-platform-tests/FileAPI/blob/Blob-bytes.any.js

```diff
@@ -0,0 +1,45 @@
+// META: title=Blob bytes()
+// META: script=../support/Blob.js
+'use strict';
+
+promise_test(async () => {
+  const input_arr = new TextEncoder().encode("PASS");
+  const blob = new Blob([input_arr]);
+  const uint8array = await blob.bytes();
+  assert_true(uint8array instanceof Uint8Array);
+  assert_equals_typed_array(uint8array, input_arr);
+}, "Blob.bytes()")
```

Comment:
Maybe, just to complement the non-ascii one below: 

```suggestion
}, "Blob.bytes() with ascii input")
```